content/docs/user-guide/how-to/stop-tracking-data.md

```diff
@@ -0,0 +1,41 @@
+# How to Stop Tracking Data
```

Comment:
Hmm ... still don't like the name :(


How would people search for this? What would be the query if we try think as an end user? Are there analogs for Git - how do they phrase this?

Reply:
The most obvious would be "remove data" but that collides with `dvc remove`... Maybe we don't really need this doc and rather an example there? The only thing that makes this how-to special is the usage of `gc`.

> This all cames from #625 BTW (now a check box in the how-tos epic).

Reply:
OR like that issue suggests, this How To could be "How to Undo/Fix (Common) Mistakes" and keep the undoing add recipe as a first one. But only if we can identify other common mistakes to add...

Reply:
Oh and in fact `remove` already has a similar example which we could just enhance with `git status` and `dvc gc`: https://dvc.org/doc/command-reference/remove#example-remove-a-tracked-file-or-directory — I'm inclining to do this and remove this whole how-to for now...

Reply:
OK I changed this to "How to Reverse Common Mistakes" although there's only one H2 section in there for now (Stop tracking data).